terms.html

```diff
@@ -169,8 +169,12 @@
 <a>verifiable presentation</a> for processing. Other specifications might refer
 to this concept as a <dfn data-lt="relying parties">relying party</dfn>.
   </dd>
-  <dt><dfn data-lt="URI|URIs">URI</dfn></dt>
+  <dt><dfn data-lt="URL|URLs">URL</dfn></dt>
   <dd>
-A Uniform Resource Identifier, as defined by [[RFC3986]].
+A Uniform Resource Locator, as defined by [[URL]]. This specification does
+not use the term URI or IRI because those terms have been deemed to be confusing
+to Web developers. This specification uses the phrase "nondereferenceable URL"
+when referring to the concept of a URI or IRI, as defined in [[?RFC3986]] and
+[[?RFC3987]], respectively.
```

Comment:
The perception of confusion is based on "search result popularity contest" [URL] from a particular search engine's users - on the entire planet.

I'd presume that the strange subset of Web developers finding URI or IRI confusing but not URL, may simply find all sorts of other things in vc-data-model to be unpenetrable.

Having said that, if URL is intended for conformance, there may not be a need to mention URI or IRI.

It may be best to not introduce phrases like "nonderefenceable URL" in vc-data-model where it may be more suitable elsewhere, if so desired, e.g., [URL].

Reply:
While I agree with the concerns you're raising, the ship sailed years ago and IETF and W3C lost control of the definition of URL to WHATWG. That's where the most up to date spec resides today, that's what the browser vendors and URL library maintainers implement, and that's what we're trying to align with in this PR.

Looks like a number of people reviewing this PR don't like the "nonderefenceable URL" language here, so I'll try to find alternate language for that.

Reply:
Fixed in eb0f404.